e2e/cases/dts/bundle/rslib.false.config.ts

```diff
@@ -0,0 +1,16 @@
+import { generateBundleCjsConfig, generateBundleEsmConfig } from '@e2e/helper';
+import { defineConfig } from '@rslib/core';
+
+export default defineConfig({
+  lib: [
+    generateBundleEsmConfig(__dirname, {
```

Comment:
The name is too long and verbose, maybe we could change it to `config.esm()` & `config.cjs()` int the future.

Reply:
In the future, we will definitely do a test related refactoring.